e2e/playwright/projects.spec.ts

```diff
@@ -1799,3 +1799,75 @@ profile001 = startProfile(sketch001, at = [0, 0])
     })
   }
 )
+
+test.describe('Project id', () => {
+  // Should work on both web and desktop.
```

Comment:
The web target is still opt-in on CI so if you explicitly want to test this in browsers, add `{ tag: ['@desktop', '@web'] }`. Otherwise, this will run on Linux Desktop, which should be fine.